pkgs/development/python-modules/locationsharinglib/default.nix

```diff
@@ -1,65 +1,53 @@
 {
   lib,
-  betamax,
   buildPythonPackage,
   cachetools,
   coloredlogs,
-  emoji,
   fetchPypi,
-  nose,
-  pythonOlder,
   pytz,
   requests,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "locationsharinglib";
   version = "5.0.3";
-  format = "setuptools";
-
-  disabled = pythonOlder "3.7";
```

Comment:
Should stay and be update to 3.9, see https://github.com/costastf/locationsharinglib/blob/5.0.3/setup.py#L49

Reply:
Fixed